packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.gts

```diff
@@ -378,22 +378,41 @@ export default class SchedulePaymentFormActionCard extends Component<Signature>
 
     this.scheduleErrorMessage = undefined;
 
-    try {
-      const { currentSafe } = this.safes;
+    const { currentSafe } = this.safes;
+
+    let payAt = this.selectedPaymentType === 'one-time' ? Math.round(this.paymentDate!.getTime() / 1000) : null;
+    let recurringDayOfMonth = this.selectedPaymentType === 'monthly' ? this.paymentDayOfMonth! : null;
+    let recurringUntil = this.selectedPaymentType === 'monthly' ? Math.round(this.monthlyUntil!.getTime() / 1000) : null
+
```

Comment:
I'm wondering why on the executePayment on hub we don't round the value, but here in the client we do, could this cause an inconsistency  ?  Maybe we could add an util on the sdk that sort of formats that so it's the same on multiple place idk

Reply:
Nice potential catch, it could be the issue. We'll see!